tests/integration.rs

```diff
@@ -141,10 +145,11 @@ fn can_init_i2c() {
 fn can_init_spi() {
     let mut sensor = new_spi(
         &[
+            SpiTrans::write(vec![Register::TEMP_CFG_REG_A, BF::TEMP_EN0 | BF::TEMP_EN1]),
             SpiTrans::write(vec![Register::CTRL_REG4_A, BF::ACCEL_BDU]),
             SpiTrans::write(vec![Register::CFG_REG_C_M, BF::MAG_BDU]),
         ],
-        default_cs(),
+        default_cs_n(2),
         default_cs(),
```

Comment:
Can you rather say `default_cs_n(3)` here?

Reply:
I wasn't sure about this change tbh. I'll fix it.

Reply:
Test fails with ```default_cs_n(3)``` for me with the following
```
---- can_init_spi stdout ----
thread 'can_init_spi' panicked at 'assertion failed: `(left == right)`
  left: `4`,
 right: `6`: Mock call number(left) and expectations(right) do not match'
```

Reply:
Ah, I see. Those are the separate assertions for the accelerometer CS and the magnetometer CS. It is correct as you wrote it.
Please disregard my comment.